### PR TITLE
Add buttons removed in #3692 with added deprecation

### PR DESCRIPTION
**Pull request by Czaki**
# Description

In this PR I would like to make 0.4.13 release as a transition release where old Qt buttons will be available but will emit warnings about deprecation. 

## Type of change

- [x] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

File: napari/_qt/widgets/qt_viewer_buttons.py [hunk 1]
```diff
@@ -385,20 +387,49 @@ def dropEvent(self, event):
             self.viewer.layers.remove_selected()
 
 
+def _omit_viewer_args(constructor):
+    @wraps(constructor)
+    def _func(*args, **kwargs):
+        if len(args) > 1 and not isinstance(args[1], str):
+            warnings.warn(
+                "viewer argument is depecated amd should not be used",
```
Comment (tlambert03) [inline napari/_qt/widgets/qt_viewer_buttons.py]:
```suggestion
                "viewer argument is deprecated and should not be used",
```

File: napari/_qt/widgets/qt_viewer_buttons.py [hunk 2]
```diff
@@ -385,20 +387,49 @@ def dropEvent(self, event):
...
+                category=FutureWarning,
+                stacklevel=2,
+            )
+            args = args[:1] + args[2:]
+        if "viewer" in kwargs:
+            warnings.warn(
+                "viewer argument is depecated amd should not be used",
```
Comment (tlambert03) [inline napari/_qt/widgets/qt_viewer_buttons.py]:
```suggestion
                "viewer argument is deprecated and should not be used",
```

**Review by tlambert03 — APPROVED** [review]
thanks! LGTM

**Comment by sofroniewn** [comment]
@Czaki can you update from main, which I think now has all the dask stuff fixed. then tests should pass and we can merge. thanks!!

**Pull request merged**